pyke/kepcotrend.py

```diff
@@ -339,22 +341,30 @@ def do_plot(date, flux_old, flux_new, bvsum, cad, good_data, cad_nans, version):
         plot_y = flux_old_sub[mask]
         if np.nan in plot_y:
             break
-        plt.plot(plot_x, plot_y, color='#0000ff', linestyle='-', linewidth=1.0)
+        plt.scatter(plot_x, plot_y, color='#363636', linestyle='-', linewidth=1.0, marker='.', s=5)
         plot_y = bvsum_sub[mask]
-        plt.plot(plot_x, plot_y, color='red', linestyle='-', linewidth=2.0)
+        plt.plot(plot_x, plot_y, color='#c0392b', linestyle='-', linewidth=2.0)
     date2 = np.insert(date_sub, [0], [date_sub[0]])
     date2 = np.append(date2, [date_sub[-1]])
     flux2 = np.insert(flux_old_sub,[0], [0.0])
     flux2 = np.append(flux2, [0.0])
-    plt.fill(date2, flux2, fc='#FFFACD', linewidth=0.0)
+    plt.fill(date2, flux2, color='#a8a7a7', linewidth=0.0, alpha=0.2, label='Data')
+    if (maskdata is None) is False:

```

Comment:
That's funny, I'm pretty sure you can do:
```
if maskdata:
```

Reply:
You're right. Doy.